packages/reports/tests/helpers/wait-for-error.js

```diff
@@ -0,0 +1,15 @@
+import { waitUntil } from 'ember-native-dom-helpers';
```

Comment:
Test helper used to wait for an error to occur.

Reply:
The helper is from this issue on ember-qunit: https://github.com/workmanw/ember-qunit-assert-helpers/issues/18

Reply:
Can we remove the assert test in
`reports/tests/integration/components/navi-date-range-picker-test.js`
and remove this file?